RecoParticleFlow/PFClusterProducer/python/pfClusterHBHEAlpaka_cff.py

```diff
@@ -0,0 +1,96 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.ProcessModifiers.alpaka_cff import alpaka
+
+from RecoParticleFlow.PFRecHitProducer.hcalRecHitSoAProducer_cfi import hcalRecHitSoAProducer as _hcalRecHitSoAProducer
+from RecoParticleFlow.PFRecHitProducer.pfRecHitHCALParamsESProducer_cfi import pfRecHitHCALParamsESProducer as _pfRecHitHCALParamsESProducer
+from RecoParticleFlow.PFRecHitProducer.pfRecHitHCALTopologyESProducer_cfi import pfRecHitHCALTopologyESProducer as _pfRecHitHCALTopologyESProducer
+from RecoParticleFlow.PFRecHitProducer.pfRecHitSoAProducerHCAL_cfi import pfRecHitSoAProducerHCAL as _pfRecHitSoAProducerHCAL
+from RecoParticleFlow.PFRecHitProducer.legacyPFRecHitProducer_cfi import legacyPFRecHitProducer as _legacyPFRecHitProducer
+from RecoParticleFlow.PFClusterProducer.pfClusterParamsESProducer_cfi import pfClusterParamsESProducer as _pfClusterParamsESProducer
+from RecoParticleFlow.PFClusterProducer.pfClusterSoAProducer_cfi import pfClusterSoAProducer as _pfClusterSoAProducer
+from RecoParticleFlow.PFClusterProducer.legacyPFClusterProducer_cfi import legacyPFClusterProducer as _legacyPFClusterProducer
+
+from RecoParticleFlow.PFClusterProducer.particleFlowCluster_cff import pfClusteringHBHEHFTask, particleFlowClusterHBHE, particleFlowRecHitHBHE, particleFlowClusterHCAL
+
+_alpaka_pfClusteringHBHEHFTask = pfClusteringHBHEHFTask.copy()
+
+pfRecHitHCALParamsRecordSource = cms.ESSource('EmptyESSource',
+            recordName = cms.string('PFRecHitHCALParamsRecord'),
+            iovIsRunNotTime = cms.bool(True),
+            firstValid = cms.vuint32(1)
+    )
+
+pfRecHitHCALTopologyRecordSource = cms.ESSource('EmptyESSource',
+            recordName = cms.string('PFRecHitHCALTopologyRecord'),
+            iovIsRunNotTime = cms.bool(True),
+            firstValid = cms.vuint32(1)
+    )
+
+pfClusterParamsRecordSource = cms.ESSource('EmptyESSource',
+            recordName = cms.string('JobConfigurationGPURecord'),
+            iovIsRunNotTime = cms.bool(True),
+            firstValid = cms.vuint32(1)
+    )
+
+hbheRecHitToSoA = _hcalRecHitSoAProducer.clone(
+        src = "hbhereco"
+    )
+
+pfRecHitHCALParamsESProducer = _pfRecHitHCALParamsESProducer.clone(
+        energyThresholdsHB = cms.vdouble( 0.1, 0.2, 0.3, 0.3 ),
```

Comment:
Why does the choice of Alpaka require us to set rec hit thresholds?

Reply:
Hmm. @jsamudio I thought these "default" values can be set with fillDescription? And, then you update them for 2023 somewhere? (and this scheme needs to be changed in a follow-up PR to take values from DB.)

Reply:
They are set by default in `fillDescriptions` in `RecoParticleFlow/PFRecHitProducer/plugins/alpaka/PFRecHitHCALParamsESProducer.cc` and here we don't actually need to set them in the configuration file unless there is a need to update them. And this is the temporary solution until the follow-up where we read from GT.

Reply:
> Why does the choice of Alpaka require us to set rec hit thresholds?

@mandrenguyen it's not the choice of Alpaka, these are the same thresholds used in the legacy modules, see [RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHE_cfi.py](https://github.com/cms-sw/cmssw/blob/master/RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHE_cfi.py#L6-L7):
```python
_thresholdsHBphase1 = cms.vdouble(0.1, 0.2, 0.3, 0.3)
_thresholdsHEphase1 = cms.vdouble(0.1, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2)
```